README.md

```diff
@@ -9,7 +9,21 @@ Paketin Seuranta is free application, no ads or paid content ever!
 You are not allowed to copy or use this app code to create similar application to 
 distribute it on Google Play or any other store.
 
-### Help required
+Table of contents
+=================
+* [Help required](#help-required)
+* [Development environment](#dev-environment)
+    * [Prerequisites](#prerequisites)
+    * [Installing](#installing)
+    * [Creating parcel logos](#creating-parcel-logos)    
+* [For participants](#for-participants)
+* [Versioning](#versioning)
+* [Author](#author)
+* [Contributors](#contributors)
+* [License](#license)
+
```

Comment:
In a work related code review I'd ask to move unrelated to changes to another PR but since this is your project and small scale, who cares!

Reply:
I should follow that approarch also here